# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/WebrtcCallWrapper.cpp
# Commit: 400d8a0f1a7d
# Full Hash: 400d8a0f1a7d5f7df39f159d4cdb3c86d9eea23b
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 2
# Description:
#   Bug 1654112 - Update conduit unittests to work with the Call worker thread. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112845
# ==============================================================================

diff -r 8d80a4ba914c -r 400d8a0f1a7d dom/media/webrtc/libwebrtcglue/WebrtcCallWrapper.cpp
--- a/dom/media/webrtc/libwebrtcglue/WebrtcCallWrapper.cpp	Mon Apr 12 14:49:20 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/WebrtcCallWrapper.cpp	Tue Apr 06 17:45:07 2021 +0200
@@ -55,11 +55,6 @@
   return wrapper;
 }
 
-/* static */ RefPtr<WebrtcCallWrapper> WebrtcCallWrapper::Create(
-    UniquePtr<webrtc::Call> aCall) {
-  return new WebrtcCallWrapper(std::move(aCall));
-}
-
 void WebrtcCallWrapper::SetCall(UniquePtr<webrtc::Call> aCall) {
   MOZ_ASSERT(mCallThread->IsOnCurrentThread());
   MOZ_ASSERT(!mCall);